contents/docs/web-analytics/marketing-analytics.mdx

```diff
@@ -58,67 +58,33 @@ The campaign breakdown table provides detailed performance metrics by individual
+PostHog can automatically sync marketing data from supported advertising platforms. You'll need to sync the required tables for each source to enable the functionality.
 
-<ProductScreenshot
-    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/native_white_0e3fb7591d.png" 
-    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/native_dark_6056080e7a.png"
-    alt="Native data warehouse sources configuration"
-    classes="rounded"
-/>
+**Supported native sources:**
+- **Google Ads**: [Setup guide](/docs/cdp/sources/google-ads) - requires `campaign` and `campaign_stats` tables
+- **LinkedIn Ads**: [Setup guide](/docs/cdp/sources/linkedin-ads) - requires `campaigns` and `campaign_stats` tables  
+- **Meta Ads**: [Setup guide](/docs/cdp/sources/meta-ads) - requires `campaigns` and `campaign_stats` tables
+- **TikTok Ads**: [Setup guide](/docs/cdp/sources/tiktok-ads) - requires `campaigns` and `campaign_report` tables
+- **Reddit Ads**: [Setup guide](/docs/cdp/sources/reddit-ads) - requires `campaigns` and `campaign_report` tables
 
-PostHog can automatically sync marketing data from support platforms like Google Ads by configuring them as native sources. You'll need to sync the required tables for each source to enable the functionality.
+Each native source automatically handles data formatting and provides the required campaign and performance data. The configuration interface shows which tables are syncing and which still need to be enabled.
 
-The simplest way to add marketing data is to [add a Google Ads source](https://app.posthog.com/pipeline/new/source?kind=googleads) by following [this guide](/docs/cdp/sources/google-ads).
+#### Data warehouse sources
 
-Required tables for native sources:
-- `campaign`: Contains campaign information and metadata
-- `campaign_stats`: Maps campaigns with performance statistics for each time period
-
-### Non-native data warehouse sources
-
-<ProductScreenshot
-    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/non_native_white_344a406529.png" 
-    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/non_native_dark_f5dd988e41.png"
-    alt="Non-native data warehouse sources configuration"
-    classes="rounded"
-/>
-
-Alternatively, you can use an external data warehouse sources to display marketing analytics. You'll need to map the required columns for each table to enable the functionality. Currently, only BigQuery tables are supported.
-
-<ProductScreenshot
-    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/configuring_white_0005e8dba1.png" 
-    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/configuring_dark_90df4b94c2.png"
-    alt="Configuring data warehouse source mapping with required and optional fields"
-    classes="rounded"
-/>
+You can connect external data warehouse sources (currently BigQuery) to display marketing analytics. You'll need to map the required columns for each table to enable the functionality.
```

Comment:
```suggestion
You can _optionally_ connect external data warehouse sources (currently BigQuery) to display marketing analytics. You'll need to map the required columns for each table to enable the functionality.
```

idk if it'll be clear enough, but at a glance it sounds like we need to. I think that's not the case